server/backend/api/app/schemas/privilege_details.py

```diff
@@ -0,0 +1,56 @@
+from typing import List, Optional
+from pydantic import BaseModel, ConfigDict, StringConstraints, model_validator
+from typing_extensions import Annotated, Literal
+
+from api.app.constants import (
+    CLIENT_NUMBER_MAX_LEN,
+    CLIENT_NAME_MAX_LEN,
+    DISTRICT_ID_MAX_LEN,
+    DISTRICT_NAME_MAX_LEN,
+    ROLE_NAME_MAX_LEN,
+)
+
+
+class ScopeSchema(BaseModel):
+    scope_type: Literal["Client", "District"]
+    client_id: Optional[
+        Annotated[str, StringConstraints(max_length=CLIENT_NUMBER_MAX_LEN)]
+    ] = None
+    client_name: Optional[
+        Annotated[str, StringConstraints(max_length=CLIENT_NAME_MAX_LEN)]
+    ] = None
+    district_id: Optional[
```

Comment:
As per above, remove the district stuff for now.

Reply:
removed district related stuff https://github.com/bcgov/nr-forests-access-management/pull/1587/commits/91166704341d868ecde8639337f807f003f70b43